### PR TITLE
Update to libwebp 1.1

### PR DESCRIPTION
**Pull request by xylar**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -82,4 +82,3 @@ extra:
     - carlodri
     - adbeda
     - oblute
-    - rlura14
```
Comment (xylar) [inline recipe/meta.yaml:85]: @adbeda, if this username was a typo, please replace with the correct username and rerender.  See https://github.com/conda-forge/tesseract-feedstock/pull/21#issuecomment-598655674

**Comment by xylar** [comment]
The pin for `libwebp` got updated here: 
https://github.com/conda-forge/conda-forge-pinning-feedstock/pull/553 
but apparently no migrator was created so we need to do it manually.

**Comment by xylar** [comment]
This isn't going to work until https://github.com/conda-forge/tesseract-feedstock/pull/24 goes in because `leptonica` has the same issue.

**Comment by CurtLH** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@CurtLH, we need to wait awhile before the new package is available.  At that point, the command will be `restart ci` rather than `rerender`.  I'll take care of it.

**Comment by xylar** [comment]
@conda-forge-admin, please restart ci

**Comment by xylar** [comment]
@conda-forge-admin, please rerender.

**Comment by xylar** [comment]
@conda-forge/tesseract, I decided to add myself as a maintainer.  I hope that's okay.  It's not that I want extra work but it seems like it might not hurt to have an extra helper.  Let me know if you prefer I remove myself, that'd be no problem.

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
@conda-forge-admin, please rerender

**Comment by xylar** [comment]
Yay, finally got this building.  @conda-forge/tesseract, could you please review and merge as soon as you are able?  And let me know if you would like me as a maintainer or not.

**Comment by CurtLH** [comment]
Thank you for your help @xylar and welcome to the team!

**Comment by carlodri** [comment]
Thanks @xylar!

**Pull request merged**